examples/dynamo_integration/scripts/throughput_analysis.py

```
@@ -0,0 +1,259 @@
+    # Group by example_number to process each workflow run separately
+    for example_num in df['example_number'].unique():
+        example_df = df[df['example_number'] == example_num].copy()
+        
+        # Sort by timestamp to ensure correct ordering
+        example_df = example_df.sort_values('event_timestamp')
+        
+        # Track LLM call boundaries
+        llm_call_id = 0
+        current_start = None
+        
+        for idx, row in example_df.iterrows():
```

Ruff (0.14.8)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)